### DISCUSSION TITLE
Accessing Storage Devices on Linux Mint

### DISCUSSION THREAD

**Discussion by iandarkwater** [freeplane/freeplane#1621] [Help]
Running on Linux Mint the system dialogs ( Save, Open etc ) don't see my devices running under  /media/myName.  I tried using symbolic links to the devices on the desktop but the dialogs don't see them either.  What am I doing wrong, please?

**Comment by fnatter** [comment]
hello iandarkwater,

Java is cross-platform and some "gui implementations" aka look-and-feels
may not know about Linux file system structure. Try to set the "look and feel" to "GTK+" or ...
in Preferences->Appearance. But for me, the default L&F "Flat IntelliJ" can access
/media. Maybe it is a permission problem?

Cheers and Best Regards,
Felix

**Reply by iandarkwater** [reply]
Hi Felix,
That's brilliant!  I was using the Nimbus 'Look & Feel', I changed to Flat IntelliJ & all is well.
I would never have suspected that solution.  Many thanks.
Best Regards
Ian
